pyopencl/__init__.py

```diff
@@ -170,6 +170,7 @@
 
 try:
     from pyopencl._cl import DeviceTopologyAmd  # noqa: F401
+    from pyopencl._cl import enqueue_copy_buffer_p2p_amd  # noqa: F401
```

Comment:
If the `DeviceTopologyAmd` import fails, your new import won't even be attempted. Is this intended?

Reply:
`DeviceTopologyAmd` and `cl_amd_copy_buffer_p2p` pretty much go hand in hand, like device topology is pretty much a hard requirement for getting p2p info.